bin/src/server/cli.rs

```diff
@@ -0,0 +1,107 @@
+use clap::{App, Arg, SubCommand};
+
+pub struct CliArgs {
+    pub port: u16,
+    pub database_url: String,
+    pub max_open_files: Option<i32>,
+    pub bulk_load_opt: Option<bool>,
+    pub sled_compression: Option<String>,
+}
+
+const PORT: &str = "PORT";
+const DATABASE_URL: &str = "DATABASE_URL";
+const ROCKSDB_MAX_OPEN_FILES: &str = "ROCKSDB_MAX_OPEN_FILES";
+const ROCKSDB_BULK_LOAD_OPTIMIZED: &str = "ROCKSDB_BULK_LOAD_OPTIMIZED";
+const SLED_COMPRESSION: &str = "SLED_COMPRESSION";
+
+const DEFAULT_PORT: u16 = 27615;
+
+pub fn parse_cli_args() -> CliArgs {
+    let database_url_argument = Arg::with_name(DATABASE_URL)
+        .help("Database url")
+        .required(true)
+        .default_value("memory://")
+        .index(1);
```

Comment:
Making it a URL is redundant with subcommands. Instead, this argument should just take a path, e.g.:

```bash
indradb sled path/to/foo.sled
```